src/test/java/io/github/pmckeown/dependencytrack/policy/PolicyAnalyserTest.java

```diff
@@ -0,0 +1,88 @@
+package io.github.pmckeown.dependencytrack.policy;
+
+import io.github.pmckeown.util.Logger;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnitRunner;
+
+import java.util.List;
+
+import static io.github.pmckeown.dependencytrack.finding.ComponentBuilder.aComponent;
+import static io.github.pmckeown.dependencytrack.policy.PolicyConditionBuilder.aPolicyCondition;
+import static io.github.pmckeown.dependencytrack.policy.PolicyViolationBuilder.aPolicyViolation;
+import static io.github.pmckeown.dependencytrack.policy.PolicyViolationListBuilder.aListOfPolicyViolations;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+import static org.mockito.Mockito.verify;
+
+@RunWith(MockitoJUnitRunner.class)
+public class PolicyAnalyserTest {
+
+    @InjectMocks
+    private PolicyAnalyser policyAnalyser;
+
+    List<PolicyViolation> policyViolations;
+    boolean isPolicyBreached;
```

Comment:
Better to define these in each method where they are used so to avoid the possibility of one test method sharing state from another.